src/types/user.ts

```diff
@@ -21,17 +21,25 @@ import { isSpidL } from "./spidLevel";
 import { SessionToken, WalletToken } from "./token";
 
 // required attributes
-export const User = t.interface({
-  created_at: number,
-  family_name: string,
-  fiscal_code: FiscalCode,
-  name: string,
-  session_token: SessionToken,
-  spid_email: EmailAddress,
-  spid_level: SpidLevel,
-  spid_mobile_phone: NonEmptyString,
-  wallet_token: WalletToken
-});
+export const User = t.intersection([
+  t.interface({
+    created_at: number,
```

Comment:
actually I just realized this is wrong, the basic types should be io-ts types (`number` -> `t.number`, `string` -> `t.string`, etc...)

Reply:
Ok I'll change it.